xrpl/models/requests/ledger_entry.py

```diff
@@ -43,6 +44,24 @@ class LedgerEntryType(str, Enum):
     NFT_OFFER = "nft_offer"
 
 
+@require_kwargs_on_init
+@dataclass(frozen=True, **KW_ONLY_DATACLASS)
+class Credential(BaseModel):
+    """Specify the Credential to retrieve. If a string, must be the ledger entry ID of
+    the entry, as hexadecimal. If an object, requires subject, issuer, and
+    credential_type sub-fields.
+    """
+
+    subject: str = REQUIRED  # type: ignore
+    """The account that is the subject of the credential."""
+
+    issuer: str = REQUIRED  # type: ignore
+    """The account that issued the credential."""
+
+    credential_type: str = REQUIRED  # type: ignore
+    """The type of the credential, as issued."""
```

Comment:
Subject isn't listed in the Credential ID object in the spec, so should this just be subject and issuer?

Reply:
hey @achowdhry-ripple , just to close the loop on this one: here's the docs for the `ledger_entry` command: https://opensource.ripple.com/docs/xls-70d-credentials/reference/ledger_entry#get-credential-entry